### PR TITLE
Add Card border

### LINKED ISSUE #1776: Card: Border updates on default variant
Default card needs border. 

"MTK have flagged that the Card is missing it's border."

FYI @joshwooding, @parsakhan21, @bhoppers2008, @libertymayc I moved the Card issue back to "In Progress" column to address this. TY!

### PR DESCRIPTION
**Pull request by libertymayc**
Design change adds border back to default Card for accessibility
closes #1776

### CODE REVIEW

File: .changeset/mighty-feet-approve.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/core": minor
```
Comment (joshwooding) [inline .changeset/mighty-feet-approve.md:2]: What type of change is this?

It sounded like a bug fix to me but not sure

Comment (libertymayc) [inline .changeset/mighty-feet-approve.md:2]:
```suggestion
"@salt-ds/core": patch
```

File: .changeset/mighty-feet-approve.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/core": patch
+---
+
+Card (default) design changed to include border
```
Comment (joshwooding) [inline .changeset/mighty-feet-approve.md:5]: Can you include why it was changed please.

Comment (libertymayc) [inline .changeset/mighty-feet-approve.md:5]:
```suggestion
Card (default) design changed to include border to enhance visual accessibility
```

Comment (libertymayc) [reply .changeset/mighty-feet-approve.md:5]: is this ok?

**Review by feliciackh — DISMISSED** [review]

File: .changeset/mighty-feet-approve.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/core": patch
+---
+
+Card (default) design changed to include border to enhance visual accessibility
```
Comment (joshwooding) [inline .changeset/mighty-feet-approve.md:5]:
```suggestion
The default Card design has been changed to include a border to enhance visual accessibility.
```

**Review by alycrys — APPROVED** [review]

**Pull request merged**